app/src/main/res/values/strings.xml

```diff
@@ -516,14 +516,14 @@
     <string name="website">Website</string>
     <string name="website_message" translatable="false">https://muntashirakon.github.io/AppManager</string>
     <string name="pref_remove_all_rules">Remove all rules</string>
-    <string name="pref_remove_all_rules_msg">All configured blocked components will be enabled, app ops will be set to default values and permissions will be granted</string>
+    <string name="pref_remove_all_rules_msg">Apps you have blocked won't be, and will instead have default persmissions and values.</string>
```

Comment:
This is still problematic. Currently, there are three things that are reverted when using this operation: app components (not the app itself), app ops and permissions.
1. App components (not apps themselves) are set to their _default_ state
2. App ops are set to their _default_ values
3. Permissions will be granted (and not set to their default values)

Reply:
Oh, you have to escape the quotations. So, `'` should be `\'`.